src/Entity/Space.php

```diff
@@ -109,4 +111,27 @@ public function setSlots(ArrayCollection $slots): void
     {
         $this->slots = $slots;
     }
+
+    public function nextIdentity(): UuidInterface
+    {
+        return Uuid::uuid4();
+    }
+
+    public function __clone()
```

Comment:
```suggestion
    public function __clone(): void
```

Reply:
clone method may not declare return types

Reply:
oh yeah like __construct, ok :)